crates/cubecl-core/src/frontend/topology.rs

```diff
@@ -134,6 +166,39 @@ constant!(
 The cube position along the Z axis.
 "
 );
+
+constant!(
+    CUBE_POS_CLUSTER,
+    crate::ir::Builtin::CubePosCluster,
+    r"
+The cube position within the cluster.
+"
+);
+
+constant!(
+    CUBE_POS_CLUSTER_X,
+    crate::ir::Builtin::CubePosClusterX,
+    r"
+The cube position in the cluster along the X axis.
+"
+);
+
+constant!(
+    CUBE_POS_CLUSTER_Y,
+    crate::ir::Builtin::CubePosClusterY,
+    r"
+The cube position in the cluster along the Y axis.
+"
+);
+
+constant!(
+    CUBE_POS_CLUSTER_Z,
```

Comment:
I would use the prefix, so `CUBE_CLUSTER_POS_X`

Reply:
I'm actually not sure that's the best option, it kind of makes it sound like it's the position of the `cubecluster` within the grid, not the position of the cube in the cluster.

Reply:
This naming is also consistent with `UNIT_POS_PLANE`.